picard/plugin.py

```diff
@@ -213,8 +213,12 @@ def license_url(self):
 
     @property
     def files_list(self):
-        return self.file[len(self.dir)+1:]
-
+        path = os.path.join(USER_PLUGIN_DIR,self.module_name)
+        path = path + '.zip'
+        if os.path.isfile(path):
+            return ', '.join(zipfile.ZipFile(path).namelist())
```

Comment:
Imho, since the method is called `files_list()`, we should always return a list:

```python
if os.path.isfile(path):
    return zipfile.ZipFile(path).namelist()
else:
    return [self.module_name + '.py']
```

And `join()` at display time. It will be easier, if, for example, we want to change from a comma-separated list to a list view or whatever.